helpers/formatVotes.ts

```diff
@@ -86,7 +86,7 @@ export function parseVoteStringWithPrecision(vote: string, decodedIdentifier: st
   return parseFixed(vote, identifierPrecision).toString();
 }
 
-export function formatVoteStringWithPrecision(vote: string, decodedIdentifier: string) {
+export function formatVoteStringWithPrecision(vote: BigNumberish, decodedIdentifier: string) {
```

Comment:
this needed to be changed because of the zero values being ignored bug mentioned earlier